scripts/index.js

```diff
@@ -0,0 +1,133 @@
+const state = {
+    tempCount: 70
+};
+
+const increaseTemp = () => {
+    state.tempCount += 1
+    const count = document.querySelector("#temperature");
+    count.textContent = state.tempCount;
+
+    if (state.tempCount > 79) {
+        red();
+        hot();
+    } else if (state.tempCount > 69) {
+        orange();
+        mild();
+    } else if (state.tempCount > 59)  {
+        yellow();
+        regular();
+    } else if (state.tempCount > 49) {
+        green();
+        cool();
+    } else {
+        blue();
+        cold();
+    };
+};
+
+const decreaseTemp = () => {
+    state.tempCount -= 1
+    const count = document.querySelector("#temperature");
+    count.textContent = state.tempCount;
+
+    if (state.tempCount > 79) {
+        red();
+        hot();
+    } else if (state.tempCount > 69) {
+        orange();
+        mild();
+    } else if (state.tempCount > 59)  {
+        yellow();
+        regular();
+    } else if (state.tempCount > 49) {
+        green();
+        cool();
+    } else {
+        blue();
+        cold();
+    };
```

Comment:
This is a repetition of the code above, so we could refactor it into a helper that we call from both places!